packages/web-components/src/components/va-radio-option/test/va-radio-option.e2e.ts

```diff
@@ -23,7 +23,7 @@ describe('va-radio-option', () => {
   it('fires event for parent when changed', async () => {
     const page = await newE2EPage();
     await page.setContent(
-      '<va-radio-option aria-describedby="test" label="A label" value="something" />',
+      '<va-radio-option checked aria-describedby="test" label="A label" value="something" />',
```

Comment:
Is this related?

Reply:
@bkjohnson not directly but since I was messing with `va-radio` I figured I'd make this fix it solves this issue with us getting the `console.warn` during the `va-radio-option` tests
Prior to fix:
<img width="929" alt="Screen Shot 2022-02-01 at 11 30 29 AM" src="https://user-images.githubusercontent.com/11822533/152009453-21ffe6f7-bcc7-4511-86c8-987325ef9907.png">
Post fix:
<img width="754" alt="Screen Shot 2022-02-01 at 11 32 08 AM" src="https://user-images.githubusercontent.com/11822533/152009750-c64dfae4-b8ea-478f-be58-b08c44fa7a4b.png">